src/main/java/FindCommand.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Represents a command to search for tasks containing a specified keyword,
+ * to be executed later.
+ */
+public class FindCommand extends Command {
+    private String keyword;
+
+    /**
+     * Constructor for FindCommand that takes in a search keyword.
+     * @param keyword the search keyword
+     */
+    public FindCommand(String keyword) {
+        this.keyword = keyword;
+    }
+
+    /**
+     * Finds the tasks containing the keyword, and calls the UI to display them.
+     * @param tasks the task list to operate on
```

Comment:
Might want to standardise leaving a line between the overall description and the @param